sub-projects/device/src/AWSClass.py

```diff
@@ -0,0 +1,45 @@
+import boto3
+import json
+
+#this class is for telescreen_device activity
+class AWS(object):
```

Comment:
This class needs a doc string below its definition, see https://google.github.io/styleguide/pyguide.html#Comments